src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1031,7 +1019,7 @@ public void onPlayerDamageByPlayer(final EntityDamageByEntityEvent e) {
 
 	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
 	public void onPlayerDeath(final PlayerDeathEvent event) {
-		if (!(event.getEntity() instanceof Player)) {
+		if (!(event.getEntity().getType() == EntityType.PLAYER)) {

```

Comment:
Just get rid of this, the entity type will obviously be a player its a player event.

Reply:
https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/entity/PlayerDeathEvent.html#getEntity()

Reply:
Yeah I agree with Vahagn